test/sandbox_test.py

```diff
@@ -319,6 +319,20 @@ def test_sandbox_exec_wait(app, servicer, exec_backend):
     assert cp.poll() == 42
 
 
+@skip_non_subprocess
+def test_sandbox_status(app, servicer):
+    sb1 = Sandbox.create(app=app, timeout=300)
+    assert sb1.status() == SandboxStatus.RUNNING
+    sb1.terminate()
+    # TODO(lucy): Figure out why this shows as failed vs. terminated
+    assert sb1.status() == SandboxStatus.FAILURE
+    # assert sb1.status() == SandboxStatus.TERMINATED
+
+    sb2 = Sandbox.create("sleep", "1", app=app, timeout=300)
+    sb2.wait()
+    assert sb2.status() == SandboxStatus.SUCCESS
+
+
 @mock.patch("modal.sandbox.CONTAINER_EXEC_TIMEOUT_BUFFER", 0)
 @skip_non_subprocess
```

Comment:
Flagging that this test - and the corresponding "timeout" test won't run - is there anything weird with timeouts in unit tests?

```python
@skip_non_subprocess
def test_sandbox_status_timeout(app, servicer):
    sb = Sandbox.create(app=app, timeout=10, idle_timeout=1)
    time.sleep(5)
    assert sb.status() == SandboxStatus.IDLE_TIMEOUT
```